docs/install-pro.md

```diff
@@ -0,0 +1,68 @@
+# Install Pro packages of Percona Server for MySQL
```

Comment:
Please add the list of packages that are available for PRO:
packages deb:
      - percona-server-server-pro
      - percona-server-test-pro
      - percona-server-client-pro
      - percona-server-pro-dbg
      - percona-server-pro-source
      - percona-server-rocksdb-pro
       -percona-mysql-router-pro
       -[percona-server-pro-common]
packages rpm:
    percona-server-server-pro
    percona-server-shared-pro
    percona-server-client-pro
    percona-server-pro-debuginfo
    percona-server-rocksdb-pro
    percona-server-test-pro
    percona-server-devel-pro
    percona-mysql-router-pro